packages/bot/src/scratch/utils.js

```diff
@@ -12,18 +12,75 @@ export const oppositesToDropdownOptions = opposite_name => {
 
 export const cleanUpOnLoad = (blocks_to_clean, drop_event) => {
```

Comment:
As per our discussion, please update the existing `cleanUp` function in `workspace_svg.js`. You can pass in the initial `cursor_x` and `cursor_y` as arguments. To not break other code that may use the function, set the default args to be `0` and `0` respectively.